cypress/TestCases/FireboltCertification/SecondScreen.feature

```diff
@@ -5,31 +5,31 @@ Feature: SecondScreen
-   @sdk @transport
-   Scenario Outline: SecondScreen.<Method> - Positive Scenario: <Scenario>
+   @sdk @transport @Sev1
+   Scenario Outline: SecondScreen.<Method> - Validating API Method responses when <Scenario>
       When '3rd party app' invokes the 'Firebolt' API to '<API_key>'
       Then 'Firebolt' platform responds with '<Validation_key>'      
 
 
       Examples:
-         | Scenario                                    | Method    | API_key                                  | Validation_key                  |
-         | Validate passing device with empty array    | device    | get secondscreen device with empty array | expected secondscreen device    |
-         | Validate passing device with string         | device    | get secondscreen device with string      | expected secondscreen device    |
-         | Validate passing protocols with empty array | protocols | get secondscreen protocols               | expected secondscreen protocols |
+         | Scenario                           | Method    | API_key                                  | Validation_key                  |
+         | passing device with empty array    | device    | get secondscreen device with empty array | expected secondscreen device    |
+         | passing device with string         | device    | get secondscreen device with string      | expected secondscreen device    |
+         | passing protocols with empty array | protocols | get secondscreen protocols               | expected secondscreen protocols |
 
-   @sdk @transport
-   Scenario Outline: SecondScreen.device - Negative Scenario: <Scenario> expecting error
+   @sdk @transport @Sev2
+   Scenario Outline: SecondScreen.device - Validating API Error handling when <Scenario>
       When '3rd party app' invokes the 'Firebolt' API to '<API_key>'
       Then 'Firebolt' platform responds with '<Validation_key>'
 
       Examples:
          | Scenario                           | API_key                              | Validation_key                             |
-         | Validate passing device as boolean | get secondscreen device with boolean | invalid parameters for secondscreen device |
-         | Validate passing device as integer | get secondscreen device with integer | invalid parameters for secondscreen device |
+         | passing device as boolean | get secondscreen device with boolean | invalid parameters for secondscreen device |
+         | passing device as integer | get secondscreen device with integer | invalid parameters for secondscreen device |
 
-   @sdk @transport
-   Scenario: SecondScreen.FriendlyName - Positive Scenario: Validate Set friendlyName
-      Given the environment has been set up for 'Secondscreen friendlyname' tests
+   @sdk @transport @Sev2
+   Scenario: SecondScreen.FriendlyName - Validating API and Event response
+      Given we test the 'SECONDSCREEN_FRIENDLYNAME' getters and setters 'name' to 'Living Room'
       And '3rd party app' registers for the 'Firebolt' event
```

Comment:
for all core test cases, we stopped using the glue code we test the 'SECONDSCREEN_FRIENDLYNAME' getters and setters 'name' to 'Living Room'

and instead used  Given the environment has been set up for 

can you use that here ?